bundle/src/main/java/com/adobe/acs/commons/users/impl/ProtectedAuthorizables.java

```diff
@@ -159,6 +159,7 @@ private ProtectedAuthorizables() {}
                 "acs-commons-review-task-asset-mover-service",
                 "acs-commons-httpcache-jcr-storage-service",
                 "acs-commons-manage-controlled-processes-service",
+                "acs-commons-resource-type-dispatcher-header-filter",
```

Comment:
Same; why do we need as service user? if the request cant access the resource to read the resource type, then this will never get hit?